<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigTab</class>
 <widget class="QMainWindow" name="ConfigTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConfigTable</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="shadow">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>1111</width>
      <height>701</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>524</width>
      <height>462</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="100,6,10">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,2">
       <item>
        <widget class="QTableWidget" name="tab_config"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="10,1">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QTextEdit" name="textEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="add_config">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hverticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>更新</string>
    </property>
    <addaction name="action11"/>
    <addaction name="action22"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action11">
   <property name="text">
    <string>下载更新</string>
   </property>
  </action>
  <action name="action22">
   <property name="text">
    <string>更新</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
